metis-debias/metis-debias-detect-service/src/main/java/eu/europeana/metis/debias/detect/model/response/ValueDetection.java

```diff
@@ -0,0 +1,70 @@
+package eu.europeana.metis.debias.detect.model.response;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The type Value detection.
+ */
+public class ValueDetection {
+
+  private String language;
+  private String literal;
+  private List<Tag> tags;
+
+  /**
+   * Gets language.
+   *
+   * @return the language
+   */
+  public String getLanguage() {
```

Comment:
Also here: perhaps mention that the 2-letter ISO code is returned here? Same for the setter.

Reply:
done.